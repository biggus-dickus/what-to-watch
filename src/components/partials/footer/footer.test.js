import * as React from 'react';
import renderer from 'react-test-renderer';

import Footer from './footer';

const tree = renderer.create(<Footer pathname="/" />).toJSON();

it(`<Footer /> should render correctly`, () => expect(tree).toMatchSnapshot());
