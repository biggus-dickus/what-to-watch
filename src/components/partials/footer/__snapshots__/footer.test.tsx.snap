// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  className="page-footer"
>
  <div
    className="logo"
  >
    <span
      className="logo__link logo__link--light"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </span>
  </div>
  <div
    className="copyright"
  >
    <p>
      Â© 
      2019
       What to watch Ltd.
    </p>
  </div>
</footer>
`;
